pkg/inspector/image-inspector_ginkgo_test.go

```diff
@@ -0,0 +1,171 @@
+// +build integrationtest
+
+package inspector_test
+
+import (
+	"fmt"
+	"github.com/fsouza/go-dockerclient"
+	. "github.com/onsi/ginkgo"
```

Comment:
we decided in origin to not use '.' imports because they are dangerous. it seems ok here, because this is a small project, but use "g" and "o" is what we do in origin extended tests :-)